specs/turborepo/run-filtered-pipeline.yml

```diff
@@ -0,0 +1,13 @@
+name: Run a Turborepo pipeline with a package filter
+command: npx turbo run {{workflows}} --filter={{filter}}
+tags:
+  - turborepo
+arguments:
+  - name: workflows
+    description: The workflows to run.
```

Comment:
nit: could we refer to the argument that's passed to `npx turbo run` as `tasks` instead of workflows (in the workflow itself and in the documentation). That's how turborepo references this argument, and I think that will avoid the name confusion with Warp workflows

<img width="681" alt="image" src="https://user-images.githubusercontent.com/4110292/163816905-647ec702-f0ae-4330-aaf3-579f37483ee3.png">

Reply:
With `turbo run`, you run a pipeline which in turn runs tasks ([Pipelines | Turborepo](https://turborepo.org/docs/features/pipelines)).